components/esp_lvgl_port/include/esp_lvgl_port_disp.h

```diff
@@ -80,7 +80,9 @@ typedef struct {
  * @brief Configuration MIPI-DSI display structure
  */
 typedef struct {
-    int dummy;
+    struct {
+        unsigned int avoid_tearing: 1;  /*!< 1: Use internal RGB buffers as a LVGL draw buffers to avoid tearing effect */
```

Comment:
Is there any case when users would need to set this to 0? From the description it seems like it should always be set to 1. In which case, does it need to be configurable?

Reply:
In fact, enabling tear prevention means frame synchronization is required, which can impact the frame rate. Additionally, enabling tear prevention will consume more memory resources, and for some users with relatively simple UI scenarios, this feature may not be necessary. Therefore, I believe it's important to keep this option available.

Reply:
Great, thanks for explaining. Can you mention some of these cons in the description of this field? E.g. "enabling this option increases RAM usage and reduces frame rate."

Reply:
Screen tearing occurs when reading from and writing to the same frame buffer simultaneously, resulting in old and new frames appearing on the same screen. This creates a noticeable tear for the human eye. To address this issue for RGB screens and MIPI-DSI interface screens, we typically use multiple frame buffers to implement anti-tearing operations. This requires allocating an additional 1 to 2 full-screen buffers. For example, with a screen resolution of 1024 x 600 on the P4 development board and using the RGB565 color format, this results in an additional PSRAM memory usage of 1M to 2M.

As mentioned earlier, we use a multi-buffer approach to avoid tearing, which requires buffer switching and frame synchronization (for instance, we need to wait until the screen has completely read a frame of data before switching frame buffers). This is inherently more time-consuming than not using anti-tearing features (where frame buffers are continuously rendered without synchronization), leading to a certain degree of frame rate drop. However, for some users with simple UI scenarios (e.g., screens that change infrequently), it is likely that they can meet their needs without utilizing this anti-tearing feature.

Reply:
I will describe these drawbacks in the comments later. Thank you very much for reviewing this part!